packages/playground/website/public/logger.php

```diff
@@ -0,0 +1,92 @@
+<?php
+header('Content-Type: application/json');
+$channel = getenv('SLACK_CHANNEL');
+$token = getenv('SLACK_TOKEN');
+
+/**
+ * Send response to the client
+ *
+ * @param bool $ok - If the request was successful
+ * @param string|null $error - Error message if the request was not successful
+ */
+function response($ok, $error = null)
+{
+    $response_data = array(
+        'ok' => $ok,
+    );
+    if ($error) {
+        $response_data['error'] = $error;
+    }
+    die(json_encode($response_data));
```

Comment:
Would the response ever contain any sensitive information we might not want to expose?

Reply:
Ah nevermind, this only exposes the explicitly passed arguments